app/web/controllers/audio_files.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import cherrypy
+from cherrypy._cprequest import _cpreqbody
+
+from app.config import Config
+from app.models import UserRole
+
+CHUNK_SIZE = 1 << 16
+
+logger = logging.getLogger(__name__)
+
+
+class AudioFiles():
+    def __init__(self):
+        self.template = Config.jinja_env.get_template('audio_files/index.html')
+
+    @cherrypy.expose
+    @cherrypy.tools.allow(methods=['GET'])
+    @cherrypy.tools.authenticate()
+    @cherrypy.tools.authorize(role=UserRole.MODERATOR)
+    def index(self):
+        files = [file.name for file in Config.audio_folder.iterdir() if file.is_file()]
```

Comment:
in the current music folder, we have the following content:
```
./sp1
./sp1/music4.mp3
./sp1/music5.mp3
./sp1/music2.mp3
./sp1/music3.mp3
./sp1/music1.mp3
./gimn.mp3
./sp3
./sp3/music4.mp3
./sp3/music5.mp3
./sp3/music2.mp3
./sp3/music3.mp3
./sp3/music1.mp3
./break
./break/break1.mp3
./break/break4.mp3
./break/break2.mp3
./break/break3.mp3
./break/break6.mp3
./break/break5.mp3
./sp2
./sp2/music4.mp3
./sp2/music5.mp3
./sp2/music2.mp3
./sp2/music3.mp3
./sp2/music1.mp3
./sport
./sport/music4.mp3
./sport/music5.mp3
./sport/music2.mp3
./sport/music3.mp3
./sport/music1.mp3
./mute_minutes.mp3
```
will this display this properly?

Reply:
No idea. Supporting nested folders was not specified in the task.